### ISSUE TITLE
Bug: Some voice_assets are not loaded correctly.

### ISSUE THREAD

**Issue by StefanTiess** [heremaps/here-sdk-examples#181]
Hi, are you aware of an Issue for Voice Guidance in SDK 4.12.7 ?: Some Languages, eg LanguageCode.DE_DE, are failing to load from the assets folder.

Even though the supportedVoiceSkins contains the Language Code in question, whenever a Maneuver comes up I get the following Error in the Logs:

`[ERROR] notification_generation - get_voice_package_output( ) - Error generating notification
AndroidAssetsLoader   ***  Failed to load asset stream voice_assets/voice_package_en-US/translations/de-DE/templates.lua
      java.io.FileNotFoundException: voice_assets/voice_package_en-US/translations/de-DE/templates.lua
      at android.content.res.AssetManager.nativeOpenAsset(Native Method)`

Note: This issue only occurs for some LanguageCodes, in fact I was only able to reproduce it with German, but I only tested a handful of languages. Also the `voice_assets` folder in the aar contains the `voice_package_de-DE`

**Comment by datasun** [comment]
It seems you are using` voice_assets/voice_package_en-US/ `with a "de-DE" variant. In that case, the device will fallback to German. The HERE SDK contains no support for German(US) - this can happen when a device is set to English, but is used in a Germany. If you switch the device language properly to German, the error log should disappear.

**Comment by StefanTiess** [comment]
Hi datasun, I am not sure how to prevent this: The language code passed to the `ManeuverNotificationOptions` in the `visualNavigator` is `DE_DE`. Also the Phone is set to German in Germany (Locale is `de_DE`). As far as I know this is the only way to assign the language of voice instructions. Still it tries to look for the voice package as a variant for en_US instead of the `voice_package_de_DE` that is installed. Also, as I said, it works fine for other languages, eg Spanish.

**Comment by datasun** [comment]
Reopened, we will try to look into this in ticket #15015.

**Comment by devcooch** [comment]
@StefanTiess could you please share a usage example? Do you set the language before the start of navigation or after?

**Comment by devcooch** [comment]
Do you use Flutter SDK or Android SDK?

**Comment by StefanTiess** [comment]
Android SDK.

Dmitry Kochkin ***@***.***> schrieb am Mi., 19. Okt. 2022,
23:42:

**Comment by StefanTiess** [comment]
Issue seems to be resolved in 4.12.9

**Comment by StefanTiess** [comment]
After some investigation I found the issue to be reproducible in the OfflineMaps Example App:
I manipulated it by allowoing the download of multiple regions
` String swizNameInGerman = "Schweiz";
        try {
            List<InstalledRegion> list = mapDownloader.getInstalledRegions();
            if(list.size() > 0) swizNameInGerman = "Liechtenstein";
            if(list.size() > 1) swizNameInGerman = "Vatikanstadt";
            if(list.size() > 2) swizNameInGerman = "Monaco";

        } catch (MapLoaderException e) {
            e.printStackTrace();
        }`

If you kill the app after each downloaded map, reopen it, then try to download the next map, you will get the INTERNAL_ERROR.

Curiously the log shows the follwing warnings:
```
[INFO ] CatalogManager - Existing catalog found, hrn='hrn:here:data::olp-here:ocm', version=56, catalog_handle=0
[WARN ] JournalAlgorithms - IsConfigurationValid issue, dependency not found, hrn='hrn:here:data::olp-here:ocm', dependency='hrn:here:data::olp-here:ocm-japan'
WARN ] CacheUpdateManager - PrefetchMapRegions: journal configuration do not match linked catalogs configuration, catalog_handle=0
 [WARN ] CacheUpdateManager - PrefetchMapRegions: could not start map prefetch job, catalog_handle=0, prefetch_error=kBadRequest, is_cancelled=0
```
Please provide a fix asap or advice for a workaround. I see this issue causing a lot of problems for drivers on the road.
